src/components/link/link.pw.tsx

```diff
@@ -0,0 +1,420 @@
+import { expect, test } from "@playwright/experimental-ct-react17";
+import React from "react";
+import { HooksConfig } from "../../../playwright";
+import {
+  getDataElementByValue,
+  icon,
+  link,
+  tooltipPreview,
+} from "../../../playwright/components";
+import { skipLink, linkChildren } from "../../../playwright/components/link";
+import { ICON } from "../../../playwright/components/locators";
+import { CHARACTERS } from "../../../playwright/support/constants";
+import {
+  checkAccessibility,
+  containsClass,
+} from "../../../playwright/support/helper";
+import Box from "../../../src/components/box";
+import Link, { LinkProps } from "../../../src/components/link";
+import {
+  LinkComponent,
+  LinkComponentAsButton,
+} from "../../../src/components/link/components.test-pw";
+/* eslint-disable jest/valid-expect, jest/no-disabled-tests, no-unused-expressions */
+
+const testData = [CHARACTERS.DIACRITICS, CHARACTERS.SPECIALCHARACTERS];
+const testPlaywright = "test-playwright";
+
+test.describe("check props for Link component", () => {
+  testData.forEach((className) => {
+    test(`should render className using ${className} special characters`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<LinkComponent className={className} />);
```

Comment:
**nitpick(non-blocking):** I think the format QA want is a empty line after the `mount`. You'll probably need to do this for all the other tests too.

```suggestion
      await mount(<LinkComponent className={className} />);
      
```